fusion_companies/fusion_companies.md

```diff
@@ -31,7 +31,7 @@ Inspired and started from [Julien Hillairet's list of fusion startups](http://ju
 | Commonwealth Fusion Systems (CFS)           | 2018    | Cambridge, Massachusetts, USA | MCF, tokamak, HTS | 2025 [7] | https://cfs.energy/ |
 | Fuse Energy Technologies                    | 2018    | Palo Alto, California USA     | Magnetized-Liner inertial fusion (MagLIF), ICF |  |  https://www.f.energy/ |
 | Helicity Space                              | 2018    | Berkely, California, USA      | Spacecraft, helicity drive | | https://www.helicityspace.com/ |
-| Crossfield Fusion                           | 2019    | London, UK                    | Technology for providing fusion fuel | | http://crossfieldfusion.com/ |
```

Comment:
Why removing Crossfield Fusion? While I agree that there is not much news from them, their website is still online, also they seem to have responded to the _Fusion Energy Base_ survey, see https://www.fusionenergybase.com/organizations/crossfield-fusion , otherwise they would not be listed there. Furthermore, they are listed on https://pitchbook.com/profiles/company/490931-11